new-website/utils/models/get_models.py

```diff
@@ -0,0 +1,125 @@
+"""
+Python script to download all the model data csv files from the official Deepchem repo and extract the necessary data
+"""
+import os
+import json
+import dataclasses
+from json import JSONEncoder
+import subprocess
+import pandas as pd
+import numpy as np
+
+PATH = '../../deepchem/data/models/'
+REDIRECT_URL = 'https://deepchem.readthedocs.io/en/latest/api_reference/models.html#'
+CHEATSHEETS = {
+    'general': 'https://raw.githubusercontent.com/deepchem/deepchem/master/docs/source/api_reference/general_purpose_models.csv',
+    'material': 'https://raw.githubusercontent.com/deepchem/deepchem/master/docs/source/api_reference/material_models.csv',
+    'molecule': 'https://raw.githubusercontent.com/deepchem/deepchem/master/docs/source/api_reference/molecular_models.csv'
+}
+
+model_list = []
+backend_list = []
+type_list = []
+featurizer_list = []
+
+@dataclasses.dataclass
+class Model:
+    """
+    Model class to store a single model information
+
+    Parameters
+    ----------
+    name: str
+        Name of the model
+    url: str
+        redirect url to the model on the official deepchem documentation
+    category: str
+        Category of the model
+    featurizers: list
+        List of featurizers supported by the model
+    backends: list
+        List of backends supported by the model
+    types: list
+        List of types supported by the model
+    model_id: int
+        Unique id of the model
+    """
+    name: str
+    url: str
+    category: str
+    featurizers: list
+    backends: list
+    types: list
+    model_id: int
+
+class ModelListEncoder(JSONEncoder):
+    """
+    ModelEncoder class to encode the Model object to JSON
+    """
+    def default(self, o):
+        """
+        Default function to encode the Model object to JSON
+
+        Returns
+        -------
+            json: str
+        """
+        return o.__dict__
+
+def convert_to_json(data, name, is_model=False):
+    """
+    Convert the list of data to JSON format and write to a file
+
+    Parameters
+    ----------
+        data: list
```

Comment:
Minor formatting issues in the parameters

Reply:
We should add 1 sentence descriptions for each of these arguments to say what they do